includes/gateways/paypal/admin/settings.php

```diff
@@ -100,6 +102,13 @@ function register_gateway_settings( $gateway_settings ) {
 		),
 	);
 
+	$is_connected = PayPal\has_rest_api_connection();
+	if ( ! $is_connected ) {
+		$paypal_settings['paypal_settings']['tooltip_title'] = __( 'Connect with PayPal', 'easy-digital-downloads' );
+		$paypal_settings['paypal_settings']['tooltip_desc']  = __( 'Connecting your store with PayPal allows Easy Digital Downloads to automatically configure your store to securely communicate PayPal.<br \><br \>You may see \'Sandhills Development, LLC\', mentioned during the process, that is us!', 'easy-digital-downloads' );
```

Comment:
```suggestion
		$paypal_settings['paypal_settings']['tooltip_desc']  = __( 'Connecting your store with PayPal allows Easy Digital Downloads to automatically configure your store to securely communicate PayPal.<br \><br \>You may see "Sandhills Development, LLC", mentioned during the process&mdash;that is the company behind Easy Digital Downloads.', 'easy-digital-downloads' );
```
I'm not sure "that is us" is the most clarifying wording--would vote to go ahead and be pretty specific.